server/controller/src/main/resources/db/migration/v0_4_0/V0_4_0_007__add_job_pin.sql

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2022 Starwhale, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+ALTER TABLE job_info
+    ADD pinned_time DATETIME NULL;
```

Comment:
Add index for `pinned_time` will be better because of the default `order by` when listing the jobs.